fdbserver/include/fdbserver/IKeyValueStore.h

```diff
@@ -183,7 +189,16 @@ inline IKeyValueStore* openKVStore(KeyValueStoreType storeType,
                                    bool checkChecksums = false,
                                    bool checkIntegrity = false,
                                    bool openRemotely = false,
-                                   Reference<IPageEncryptionKeyProvider> encryptionKeyProvider = {}) {
+                                   Reference<AsyncVar<ServerDBInfo> const> db = {},
+                                   Optional<EncryptionAtRestMode> encryptionMode = {}) {
+	// Only Redwood support encryption currently.
+	if (encryptionMode.present() && encryptionMode.get().isEncryptionEnabled() &&
+	    storeType != KeyValueStoreType::SSD_REDWOOD_V1) {
+		TraceEvent("KVStoreTypeNotSupportingEncryption")
```

Comment:
Should we make this a Sev40 or a Sev30?

Reply:
I will change it to Sev30. We are throwing an error which will fail the storage server. Something at a higher level will catch it and print Sev40.

Reply:
I think `SevWarn` is good enough here, anyhow specific error is propagated up the stack.